client/app/js/services.js

```diff
@@ -461,17 +458,12 @@ factory("Access", ["$q", "Authentication", function ($q, Authentication) {
       });
     };
 }]).
-  factory('RTip', ['$http', '$q', '$filter', 'RTipResource', 'RTipReceiverResource', 'RTipMessageResource', 'RTipCommentResource', 'RTipIdentityAccessRequestResource',
-          function($http, $q, $filter, RTipResource, RTipReceiverResource, RTipMessageResource, RTipCommentResource, RTipIdentityAccessRequestResource) {
+  factory('RTip', ['$http', '$q', '$filter', 'RTipResource', 'RTipMessageResource', 'RTipCommentResource', 'RTipIdentityAccessRequestResource',
+          function($http, $q, $filter, RTipResource, RTipMessageResource, RTipCommentResource, RTipIdentityAccessRequestResource) {
     return function(tipID, fn) {
       var self = this;
 
       self.tip = RTipResource.get(tipID, function (tip) {
-        tip.receivers = RTipReceiverResource.query(tipID);
-        tip.comments = tip.enable_comments ? RTipCommentResource.query(tipID) : [];
-        tip.messages = tip.enable_messages ? RTipMessageResource.query(tipID) : [];
-        tip.iars = tip.identity_provided ? RTipIdentityAccessRequestResource.query(tipID) : [];
-
         $q.all([tip.receivers.$promise, tip.comments.$promise, tip.messages.$promise, tip.iars.$promise]).then(function() {
```

Comment:
Line 467 here: `$q.all([tip.receivers.$promise. . . .` needs to be updated (the promises are outdated).